onnxruntime/core/providers/coreml/builders/op_builder.h

```diff
@@ -3,11 +3,20 @@
 
 #pragma once
 
+#include "core/graph/graph_viewer.h"
+
 namespace onnxruntime {
 namespace coreml {
 
 class ModelBuilder;
 
+struct OpBuilderInputParams {
```

Comment:
>OpBuilderInputParams [](http://example.com/codeflow?start=7&length=20)

if all we need is a const GraphViewer&, maybe just pass that to IsOpSupported instead?
if we plan to keep adding to this later, it makes sense to have a params struct. with the current name, it's not obvious that this is input for IOpBuilder::IsOpSupported().

Reply:
I was doing that,too. But per Guoyu's suggestion, since we already modified the interface this time and involved a lot of files, maybe we can add the struct so next time we won't need to modify this many functions, just adding extra struct members will be fine.  
I can try to come up with a better name.